app/views/user_mailer/ranking.html.haml

```diff
@@ -4,17 +4,21 @@
   Congratulations on completing the MITRE CTF! Your team, #{@user.team.team_name} came ranked #{@rank.ordinalize}.
 - if (@rank.eql? 1) && @team.eligible?
   %p
-    Because you finished on the first-place team in the #{@div.name} Division, you could be eligible for a scholarship and
-    consideration for an internship at MITRE or an industry partner.  Please email
-    #{link_to 'ctf@mitre.org', 'mailto:ctf@mitre.org'} with your official
+    Because you finished on the first-place team in the #{@div.name} Division, you could be eligible for a scholarship.  Please email
+    #{link_to 'ctf@mitre.org', 'mailto:ctf@mitre.org'} with your unofficial
     college, university, or high school transcript AND an up-to-date resume to verify your eligibility.
     Both a transcript and a resume are required for the award of any scholarship prize.
-- elsif (@rank <= 10) && @team.eligible?
+- if (@team.score > 0) && @user.interested_in_employment?
   %p
-    Because you finished in the top ten, we are requesting your resume and transcript. If you’re interested, please
-    email #{link_to 'ctf@mitre.org', 'mailto:ctf@mitre.org'} with your official or unofficial college, university, or
-    high school transcript AND an up-to-date resume.
-- if @rank <= 10
+    If you are interested in working at MITRE, please apply using the link below which best matches your situation:
+  %p
+    #{link_to 'I am a high school or college student and am seeking a software engineering internship position.', 'https://example.com/placeholder'}
+  %p
+    #{link_to 'I am a high school or college student and am seeking a cyber security internship position.', 'https://example.com/placeholder'}
+  %p
+    #{link_to 'I am about to graduate or have already graduated college and am seeking a full-time software engineering position.', 'https://example.com/placeholder'}
```

Comment:
Change to "looking for full-time software/cyber position"

Reply:
Done